libsql-client/src/cell_value.rs

```diff
@@ -0,0 +1,57 @@
+//! `CellValue` represents libSQL values and types.
+//! Each database row consists of one or more cell values.
+
+/// Value of a single database cell
+#[derive(Clone, Debug)]
+pub enum CellValue {
+    Text(String),
+    Float(f64),
+    Number(i64),
+    Bool(bool),
+    Null,
+}
```

Comment:
We are missing a variant for a SQLite blob.

Reply:
On purpose, its serialization wasn't done during the hackathon, when this driver was born :innocent: It should definitely be there